lib/beacon_web/live/page_management/page_live/index.html.heex

```diff
@@ -31,7 +31,7 @@ Last reload time: <%= @last_reload_time %>ms
         <td><%= page.template %></td>
 
         <td>
-          <span><%= live_patch "Edit", to: "page_editor/#{page.id}" %></span>
+          <span><%= live_redirect "Edit", to: "page_editor/#{page.id}" %></span>
```

Comment:
`live_patch` was behaving oddly, probably since we're not linking within the same live view. I think we're supposed to use `live_redirect` for this.